google-cloud-logging/.rubocop.yml

```diff
@@ -9,6 +9,7 @@ AllCops:
     - "Rakefile"
     - "support/**/*"
     - "test/**/*"
+    - "lib/google/cloud/logging/rails.rb"
```

Comment:
Good point. I did attempt to reduce the size of the method but nothing I could think of was materially better.